sdk/cognitiveservices/cognitiveservices-luis-runtime/rollup.config.js

```diff
@@ -18,16 +21,14 @@ const config = {
       "@azure/ms-rest-azure-js": "msRestAzure"
     },
     banner: `/*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- *
- *
```

Comment:
What was the reason for removing the copyright here?

Reply:
It was removed when I autogenerated it. Will add them manually.